llmware/library.py

```diff
@@ -713,7 +1115,18 @@ def add_image(self, input_folder=None):
 
     def add_pdf_by_ocr(self, input_folder=None):
 
-        """ Alternative method to ingest PDFs that are scanned, or can not be otherwise parsed """
+        """ Alternative method to ingest PDFs that are scanned, or can not be otherwise parsed 
```

Comment:
Delete space before you start the doc and add full stop.
```suggestion
        """Alternative method to ingest PDFs that are scanned, or can not be otherwise parsed.
```